src/webserver/WebServer.jl

```diff
@@ -323,6 +324,7 @@ function run(session::ServerSession, pluto_router)
         end
     end
 end
+precompile(run, (ServerSession, HTTP.Router))
```

Comment:
```suggestion
```
You can test the precompile directive in the REPL to be sure that it works. This one doesn't:

```julia
julia> precompile(Pluto.run, (Pluto.ServerSession, HTTP.Router))
false
```
because not all types are concrete
```julia
julia> isconcretetype(HTTP.Router)
false
```